src/hitch.c

```diff
@@ -3985,8 +3992,12 @@ main(int argc, char **argv)
 		exit(1);
 	}
 
-	if (geteuid() == 0 && CONFIG->UID < 0) {
-		ERR("{core} ERROR: Refusing to run workers as root.\n");
+	__uid_t denes = geteuid();
+	(void)denes;
```

Comment:
DENES!

Reply:
Is the multiple `geteuid` necessary? To me it seems like the `denes` variable can be removed entirely, and then the `__uid_t` can go as well.

Reply:
Damn. New low. I forgot debug code in the PR :(

Reply:
I was unsure if at loglevel=0 ERR("{core} ERROR: Refusing...") not logging is expected or not.